lib/cc/analyzer/formatters/formatter.rb

```diff
@@ -2,6 +2,8 @@ module CC
   module Analyzer
     module Formatters
       class Formatter
+        attr_reader :current_engine
```

Comment:
Ruby's method security is weird and that's not what `protected` means. But it could be `private`, sure.